docs/ref-manual/cli/options.md

```diff
@@ -130,6 +132,15 @@ An example of an “empty” rule:
 
 _We expect all rules to fail this check._ The exception is the fallback function, which might pass.
 
+2. Assert-Vacuity- checks for each assert command in the rule, whether the assert is vacuously true, namely, even if all the previous preconditions are removed, the assert is always evaluated to true.
```

Comment:
I would split this into two sentences. "...vacuously true.  A rule is vacuously true if ..."